### PR TITLE
MM-27454 - Contact Us and Billing Documentation Links

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary

- Changed `Contact Support` and `Contact Sales` links to point to the [new Cloud Contact us](https://github.com/mattermost/customer-web-server/pull/245) page at `/cloud/contact-us` in CWS with query params for MM User full name and email. 
- Updated `How Billing works` links to new Cloud Billing documentation page [here](https://docs.mattermost.com/overview/mattermost-cloud-overview.html#how-billing-works)

#### Ticket Link
[MM-27454](https://mattermost.atlassian.net/browse/MM-27454)

#### Related Pull Requests
CWS: https://github.com/mattermost/customer-web-server/pull/245
Redux: https://github.com/mattermost/mattermost-redux/pull/1255
Server: https://github.com/mattermost/mattermost-server/pull/15916

### CODE REVIEW

File: components/admin_console/billing/billing_history.tsx [hunk 1]
```diff
@@ -29,7 +31,7 @@ const noBillingHistorySection = (
         <a
             target='_blank'
             rel='noopener noreferrer'
-            href='http://www.google.com'
```
Comment (ali-farooq0) [inline components/admin_console/billing/billing_history.tsx:32]: heh :D

File: components/purchase_modal/purchase_modal.tsx [hunk 2]
```diff
@@ -169,7 +175,11 @@ export default class PurchaseModal extends React.PureComponent<Props, State> {
                                 />
                             </span>
                             {'\u00A0'}
-                            <a href='https://support.mattermost.com/hc/en-us/requests/new?ticket_form_id=360000640492'>
+                            <a
+                                href={CloudLinks.BILLING_DOCS}
+                                target='_blank'
```
Comment (ali-farooq0) [inline components/purchase_modal/purchase_modal.tsx:180]: low priority but worth checking to see if this works on desktop

File: components/admin_console/billing/billing_subscriptions.tsx [hunk 3]
```diff
@@ -93,6 +44,66 @@ const BillingSubscriptions: React.FC<Props> = () => {
...
+                    <FormattedMessage
+                        id='admin.billing.subscription.privateCloudCard.description'
+                        defaultMessage='If you need software with dedicated, single-tenant architecture, Mattermost Private Cloud (Beta) is the solution for high-trust collaboration.'
+                    />
+                </div>
+                <button className='PrivateCloudCard__contactSales'>
+                    <a
+                        href={contactSalesLink}
+                        rel='noopener noreferrer'
+                        target='_blank'
```
Comment (ali-farooq0) [inline components/admin_console/billing/billing_subscriptions.tsx:92]: all these `_blank` <a> elements would be worth checking if they work as expected on the desktop client.

**Review by ali-farooq0 — APPROVED** [review]

File: components/admin_console/billing/billing_subscriptions.tsx (hunk 3 above)
Comment (marianunez) [reply components/admin_console/billing/billing_subscriptions.tsx]: Good catch! Fixed them all to use `target='_new'` which does work properly on both webapp and desktop 👌

**Comment by justinegeffen** [comment]
@marianunez, just to note that the cloud billing docs are not live yet (I'm sure you know that!). Do you have an ETA as to when this will be merged so we can align with the doc release?

**Comment by marianunez** [comment]
> @marianunez, just to note that the cloud billing docs are not live yet (I'm sure you know that!). Do you have an ETA as to when this will be merged so we can align with the doc release?

@justinegeffen this is expected to go live for Soft GA this Thrusday Oct 15th. Would that doc be published by then?

**Comment by justinegeffen** [comment]
Thanks, @marianunez! I've updated the doc PR and @adamjclarkson is aware. :)

File: components/admin_console/billing/plan_details.tsx [hunk 4]
```diff
@@ -67,8 +68,18 @@ const seatsAndSubscriptionDates = (locale: string, userCount: number, numberOfSe
                                     <div className='BillingSubscriptions__tooltipMessage'>
                                         <FormattedMarkdownMessage
                                             id='admin.billing.subscription.planDetails.prolongedOverages'
-                                            defaultMessage='Prolonged overages may result in additional charges. [See how billing works](!https://google.com)'
+                                            defaultMessage='Prolonged overages may result in additional charges.'
                                         />
+                                        <a
```
Comment (devinbinnie) [inline components/admin_console/billing/plan_details.tsx:73]: The markdown formatter already opens the link in a new window/tab, this link feels unnecessary.

File: components/admin_console/billing/billing_history.tsx [hunk 5]
```diff
@@ -27,9 +29,9 @@ const noBillingHistorySection = (
             />
         </div>
         <a
-            target='_blank'
+            target='_new'
```
[the remark this answers is not in the document]
Comment (marianunez) [reply components/admin_console/billing/billing_history.tsx:32]: `_blank` is currently blocked on the desktop app. @Willyfrog what do you recommend here to open links from the desktop app?

**Comment by marianunez** [comment]
@stevemudie Will be tested post-merge once https://github.com/mattermost/customer-web-server/pull/245 is complete.

File: components/admin_console/billing/billing_history.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (Willyfrog) [reply components/admin_console/billing/billing_history.tsx:32]: it shouldn't be blocked, have you checked with 4.6-rc1?

[the remark this answers is not in the document]
Comment (marianunez) [reply components/admin_console/billing/billing_history.tsx:32]: Checked in 4.6-rc1 and is good with both `_new` and `_blank`. What is the recommended one to use ?

[the remark this answers is not in the document]
Comment (Willyfrog) [reply components/admin_console/billing/billing_history.tsx:32]: I'd say `_blank`

**Pull request merged**